### ISSUE TITLE
多输入语义分割（2张输入，2张输出）训练 loss在上升。

### ISSUE THREAD

**Issue by HXia-XXL** [PaddlePaddle/Paddle#28986]
用paddle复写的之前在xx火炬运行正常的代码。任务是对两张输入影像做语义分割（两个输出图分别对应于两张输入图）
`
    import paddle.optimizer as optim
    criterion_CE = paddle.nn.loss.CrossEntropyLoss()
    
    scheduler = optim.lr.ReduceOnPlateau(learning_rate=lr, factor=0.1, patience=5, verbose=True)

    optimizer = optim.Momentum(
            parameters=net.parameters(),
            learning_rate=scheduler,
            momentum=0.9,
            weight_decay=0.0001)
`
迭代器部分就是
`
         for i_batch, (img1, img2, label1, label2, label) in enumerate(train_loader):

            label1 = paddle.cast(label1, dtype='int64')
            label2 = paddle.cast(label2, dtype='int64')

            output1, output2 = net(img1, img2)

            loss1 = criterion_CE(output1, label1)
            loss2 = criterion_CE(output2, label2)

            loss = loss1 + loss2
            loss.backward()
            optimizer.step()
            optimizer.clear_grad()
            #
            train_loss += loss.numpy()[0] 
`        
Batchsize和lr都设置得和xx火炬一样，分别为4和0.007
打印train_loss/len(train_loader)就是一个epoch的loss值：
epoch    loss
0            1.47
1            1.39
2            1.98
3            1.97
4            1.79
5            1.76
代码和数据都是从原来运行正常的xx火炬项目copy过来的。
寻找好心大神帮我康康是哪里出错了

**Comment by HXia-XXL** [comment]
问题好像是出在定义的注意力卷积层那里。可能不能直接改函数名迁移到paddle来把
